.github/workflows/release.yml

```diff
@@ -83,6 +106,12 @@ jobs:
       - name: 📥 Download deps
         run: npm install --no-package-lock
```

Comment:
Is this step required for release? It feels like no, but I also don't really want to mess with the release workflow